app/src/main/kotlin/com/hedvig/app/feature/adyen/payin/AdyenExtensions.kt

```diff
@@ -37,7 +37,7 @@ fun Activity.startAdyenPayment(market: Market?, paymentMethods: PaymentMethodsAp
     )
     .addCardConfiguration(cardConfig)
     .addGooglePayConfiguration(googlePayConfig)
-    .setShopperLocale(getLocale(this, market))
+    .setShopperLocale(getLocale())
```

Comment:
An example of using the top-level one instead of the one provided by LanguageService.
I guess it's hard to use that one in extension functions like this, but this will mean that it'll always be easier to reach for the top-level one from everywhere. Even by accident like we do in "ClaimDetailActivity".
Any thoughts on this? Maybe not the end of the world, but if we keep both I'd rather the LanguageService to also call this top level getLocale as an implementation detail so that no matter where you're coming from you're ending up in the same one implementation, to keep it simple to understand and change without having to do it in multiple places (if we ever need to do that)